src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -502,16 +544,38 @@ namespace webnn_native { namespace ie {
                 &conv2dNode);
             DAWN_TRY(CheckStatusCode(status, "ngraph group convolution"));
         } else {
-            IEStatusCode status = ngraph_convolution(
+            status = ngraph_convolution(
                 input, filterNode, strides.data(), strides.size(), padding.data(), padding.size(),
                 dilations.data(), dilations.size(), static_cast<ngraph_auto_pad>(options->autoPad),
                 &conv2dNode);
             DAWN_TRY(CheckStatusCode(status, "ngraph convolution"));
         }
+        ngraph_node_t* addNode;
+        ngraph_node_t* biasNode;
```

Comment:
Move `biasNode `to local variable  and remove `addNode`

ngraph_node_t* biasNode = const_cast<ngraph_node_t*>(mGraphNodeMap[conv2d->Inputs()[2].Get()]);

Reply:
Good suggestions, thanks! @fujunwei